### PR TITLE
Docs 54 test

### PR DESCRIPTION
**Pull request by mjang-cobalt**

### CODE REVIEW

File: content/en/Cobalt_Core/Learn/_index.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: "Learn More"
+linkTitle: "Learn More"
+weight: 3
+description: >
+  Some one-line description
+---
+
+{{% pageinfo %}}
+Some one-line summary
```
Comment (mike2jang) [inline content/en/Cobalt_Core/Learn/_index.md:10]: You need to say something else

**Review by mike2jang — APPROVED** [review]

**Review by mike2jang — CHANGES_REQUESTED** [review]
I need something else

File: content/en/Cobalt_Core/Learn/aws_cloud_test.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+title: "Amazon AWS Cloud Configuration Pentesting"
+linkTitle: "AWS Cloud Configuration Pentesting"
+weight: 80
+description: >
+  Some one-line description
+---
+
+{{% pageinfo %}}
+Some one-line summary
```
Comment (mike2jang) [inline content/en/Cobalt_Core/Learn/aws_cloud_test.md:10]: This is something else

**Review by mike2jang — APPROVED** [review]

File: content/en/Cobalt_Core/Learn/pentest_tools.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+---
+title: "Penetration Testing Tools"
+linkTitle: "Pentest Tools"
+weight: 90
+description: >
+  Some one-line description
```
Comment (mjang) [inline content/en/Cobalt_Core/Learn/pentest_tools.md:6]: This is a test

File: content/en/Cobalt_Core/Learn/pentest_tools.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+---
+
+{{% pageinfo %}}
```
Comment (mjang) [inline content/en/Cobalt_Core/Learn/pentest_tools.md:9]: A second test

File: content/en/Cobalt_Core/Onboarding/_index.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
+---
+title: "Onboarding"
+linkTitle: "Onboarding"
+weight: 5
+description: >
+  Some one-line description
+---
```
Comment (mjang) [inline content/en/Cobalt_Core/Onboarding/_index.md:7]: A third test

**Review by mjang — CHANGES_REQUESTED** [review]
test

**Review by ana-dashuk-cobalt — APPROVED** [review]
@mjang-cobalt I see that it's a test PR. Approved.

**Pull request closed without merging**